spec/models/project_spec.rb

```diff
@@ -65,6 +89,20 @@
         end
       end
     end
+
+    it "won't update a sub project" do
+      parent = FactoryBot.create(:project)
+      sub_project1 = FactoryBot.create(:project, parent: parent)
+      sub_project2 = FactoryBot.create(:project, parent: parent, status: "archived")
+
+      expect(sub_project1.toggle_archived!).to eql(nil)
+      sub_project1.reload
+      expect(sub_project1.status).to eql(nil)
```

Comment:
maybe `expect(sub_project1).not_to be_archived`?

Reply:
I think it's better to test the semantic than the actual attribute value since we already have tests for what each value of `status` means